.github/workflows/tests-docker.yml

```diff
@@ -26,6 +26,18 @@ jobs:
         docker version
         docker compose version
         python3 -V
+    - name: setup wis2downloader
+      run: |
+        # make sure current user in docker group
+        sudo usermod -aG docker $(whoami)
+        # Add env parameters to tests/test.env
+        echo "" >> tests/test.env  #no new line at end of test.env file
+        echo "DOCKER_GID=$(getent group docker | cut -d: -f3)" >> tests/test.env
+        echo "DOWNLOADER_UID=12135" >> tests/test.env
```

Comment:
as discussed DOWNLOADER_UID can not be used as an environment variable but has to be set to 12135 for the downloader-image to run